### ISSUE TITLE
Add menu to family records

### ISSUE THREAD

**Issue by Neriderc** [Neriderc/GVExport#520]
This suggestion was originally posted here:

> I agree. To effectively work with webtrees mainly from a GVExport-fullscreen-interface, clicks on families should get a pop-up-menu simular to that of individuals.

_Originally posted by @schuco in https://github.com/Neriderc/GVExport/issues/402#issuecomment-2150644200_
            
What kind of options do we want in a menu that shows when you click on a family record? We don't have many settings that affect families at the moment.

**Comment by hartenthaler** [comment]
maybe:
- remove descendants from diagram
- remove ancestors from diagram

**Comment by Neriderc** [comment]
> maybe:
> 
>     * remove descendants from diagram
> 
>     * remove ancestors from diagram

I don't think we currently have functionality to do this. We have "stopping" individuals, but this doesn't really remove ancestors or descendants and instead doesn't add any further people when crawling the tree.

I don't think I quite understand what these functions would do, that is different from stopping individuals. Anything different from stopping individuals makes me think it will break the diagram (if you remove all people linked through that person then that's just the same as stopping individuals, if you don't remove them then you get unlinked people floating around the diagram). We already have an option to add individuals to the stopping individuals list.

Did you have an idea of what would happen when these items are clicked?

**Comment by hartenthaler** [comment]
My idea was something like defining a stopping family. But that could be done already by adding wife and husband to the list of stopping individuals.

**Comment by schuco** [comment]
What about "showing selected families in a different colour"

**Comment by Neriderc** [comment]
> What about "showing selected families in a different colour"

Could do that, if it's something you'd find useful?

**Comment by hartenthaler** [comment]
A user selected color for a specific family box? I don't see a need for that. But who knows? 

It would be interesting to see different colors for different types of families, like families with or without a MARR event or for same-sex families or for families with only one known partner. But I think that is another topic.

**Comment by schuco** [comment]
> something you'd find useful?

useful but not necessary

**Comment by Neriderc** [comment]
When I come to look at doing this menu, I'll look at how easy it is to replicate this functionality to family records.
